### DISCUSSION TITLE
Custom Domain already Taken problem

### DISCUSSION THREAD

**Discussion by uthsobcb** [community/community#27626] [Other Feature Feedback, Questions, & Ideas]
I'm trying to connect my domain with a github page but there showing-
_`The custom domain uthsob.me is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain`_
but i went to doc and verified the domain and tried but it showing same error. When i go to the domain it shows githubs 404 page.
What can I do??

**Comment by jfullstackdev** [comment]
not very sure of this one, but your root should be setup first,

it should be <https://uthsobcb.github.io/> and the repo name is `uthsobcb.github.io` and this tells us, it's 404 not yet configured, 

![image](https://user-images.githubusercontent.com/47092464/181805063-bdf6b767-1bfb-45e1-bf34-f1e3d8e4e0f0.png)

try renaming your repo `uthsob.me` to `uthsobcb.github.io` ( or create one ) and remove the redirection first to see whether it will be correct, then once correct, you can now configure it again, verify and do the custom domain

because we are seeing GitHub Pages 404, it's already hosting but should be configured correctly

**Comment by uthsobcb** [comment]
sorry but what is `remove the redirection first` means? I mean where/how can I remove that? 
I cleared all configuration from my domain nameserver panel.

**Reply by jfullstackdev** [reply]
yup that's right remove first the config for the custom domain,

**Reply by uthsobcb** [reply]
i tried by removing config and add this domain but it showing same error!

**Reply by jfullstackdev** [reply]
were you able to successfully launched the root of your GitHub Pages without the custom domain ?

**Reply by uthsobcb** [reply]
Sorry what do mean by launching from root? Yes it's running without custom domain but when i want to add domain it showing this error.
![Screenshot from 2022-07-31 11-42-48](https://user-images.githubusercontent.com/51291526/182012028-eef37dec-8102-48cc-bbbc-d79b77dc0aaf.png)

**Reply by jfullstackdev** [reply]
if you already successfully launched the default one, then it's your setup, hopefully one of the Staff members will see this issue

**Comment by mpboom** [comment]
Did you try following the steps over here:
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages

as suggested by the error?

**Reply by uthsobcb** [reply]
Yes I did but same problem :)

**Comment by uthsobcb** [comment]
So Update is I couldn't figured out how to solve so I actually hosted this on [render](https://render.com/) and added the domain there.
But if anyone know how can I solve the problem actually then please answer this.

**Reply by jfullstackdev** [reply]
now it's being hosted by the default domain 

![image](https://user-images.githubusercontent.com/47092464/182826210-2b260b9b-d1a7-40b4-82fb-220a55d81154.png)

would you want to try setting it up again ?

**Reply by uthsobcb** [reply]
Well I don't wanted to use third party to host this but had to. So it would be great if it host without third-party.

**Reply by jfullstackdev** [reply]
meaning, you will not add the custom domain anymore ?

**Reply by uthsobcb** [reply]
No I meant I want to use custom domain but directly from github not by render

**Reply by jfullstackdev** [reply]
check this out 

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site
